src/ts/order.ts

```diff
@@ -168,13 +170,15 @@ export function normalizeOrder(order: Order): NormalizedOrder {
 }
 
 /**
- * Compute the 32-byte digest for the specified order.
+ * Compute the 32-byte signing has for the specified order.
+ *
+ * @param domain The EIP-712 domain separator to compute the hash for.
  * @param order The order to compute the digest for.
  * @return Hex-encoded 32-byte order digest.
  */
-export function hashOrder(order: Order): string {
-  return ethers.utils._TypedDataEncoder.hashStruct(
-    "Order",
+export function hashOrder(domain: TypedDataDomain, order: Order): string {
```

Comment:
maybe the name should changle slightly to reflect the difference in logic. How about `orderSigningHash`?